src/components/pagination-nav/pagination-nav.js

```diff
@@ -63,9 +63,15 @@ export default {
   },
   methods: {
     onClick(pageNum, evt) {
-      // Update the v-model
-      this.currentPage = pageNum
+      if (pageNum === this.currentPage) {
+        // Dont do anything if clicking the current active page
```

Comment:
Would suggest to put the comment above the if.

Reply:
Yeah...

Still playing with this to figure out what is causing the issue...

It appears that the link is getting re-rendered before the link action happens, and is therefore routing to the next page (or previous page) instead of the requested page.